### DISCUSSION TITLE
Creative search bar and REI not showing any recipe

### DISCUSSION THREAD

**Discussion by Zerely** [shedaniel/RoughlyEnoughItems#1711] [Q & A]
Whenever i search for something in the creative search bar nothing appears, and REI is not showing any recipe even when the cheat mode is on (the mod JEI solves this, but i really dont want to use JEI)
![image](https://github.com/user-attachments/assets/bb7f9090-2e73-48f1-b3c8-904ea42a14b1)

**Comment by TheLegendofSaram** [comment]
same here, only works when you reload the plugins

**Comment by Xaaali** [comment]
Happening for me too. Every time I log into a world, REI shows no items, even when searching, and I have to manually Reload Plugins in the settings to get it to display properly. Very frustrating

**Comment by iCuraCN** [comment]
find the performance in the settings，disable cached rendering and click done
![00](https://github.com/user-attachments/assets/9fad3a4e-19b2-4eed-841c-71df17de1186)
